src/renderer/components/ProgramChangeView.tsx

```diff
@@ -291,6 +282,15 @@ function ProgramChangeView() {
               <Button
                 onClick={() => {
                   const newAccount = createNewAccount();
+                  // TODO: yeah, this is horrible, will be easier to do generating the new Key on the server, and saving it there...
```

Comment:
This will change in a future PR that actually moves it - but there's wallet work I need to do first to make that worth it.